### PR TITLE
Fix "Uncaught TypeError: Cannot set property '???' of undefined" because of overwriting

### PR DESCRIPTION
**Pull request by occar421**
See Issue #41 

I suspect it is caused by 'export-path.js'. So I added one unit text and fix that.
How do you think about it? @mxmul

### CODE REVIEW

**Comment by occar421** [comment]
I'm sorry that CI environment was really different from mine...
It could take time than I expected. 😢

**Comment by occar421** [comment]
Now I think the __snapshots__ is old (saved with wrong result.)
In ./tests/es6/legacy-code/module.js
```js
goog.provide('some.namespace.submodule');
goog.require('some.namespace');
console.log('This should contain the empty submodule:', some.namespace);
```
Currently, `"This should contain the empty submodule: { value: 'This is a module' }` is saved as a snapshot.  
I assume `"This should contain the empty submodule: { value: 'This is a module', submodule: {} }` is correct now and suitable for the assertion description.

File: lib/export-path.js [hunk 1]
```diff
@@ -13,12 +13,25 @@ module.exports = function exportPath(namespace, name, value) {
...
+    var keys;
+
     for (i = 0; i < parts.length; i += 1) {
         part = parts[i];
         if (i === parts.length - 1 && value !== undefined) {
-            target[part] = value;
+            if (target[part]) {
+                prev = target[part];
+                target[part] = value;
+                keys = Object.keys(prev);
```
Comment (mxmul) [inline lib/export-path.js:28]: FYI this file is included in the loader output, so any features we use in here must be supported by the target browser. Using `Object.keys` means older browsers (like IE8) won't be able to consume Webpack bundles created using this loader unless they use a polyfill.

I think I'm OK with this though.

**Review by mxmul — APPROVED** [review]
Thanks for contributing!

File: lib/export-path.js (hunk 1 above)
Comment (occar421) [reply lib/export-path.js:28]: Thank you.
OK, that's why the coding rule seemed so restrictive for me 🙄
Fair-enough point 🙇

**Pull request merged**